src/deploy/NVA_build/builder.Dockerfile

```diff
@@ -7,14 +7,13 @@ LABEL maintainer="Liran Mauda (lmauda@redhat.com)"
 #   Size: ~ 613 MB
 #   Cache: Rebuild when we adding/removing requirments
 ##############################################################
-ENV container docker
 # RUN dnf --enablerepo=PowerTools install -y -q nasm && \
 #     dnf clean all
-RUN dnf update -y -q --nobest && \
-    dnf clean all
-RUN dnf install -y -q wget unzip which vim python2 python3 && \
-    dnf group install -y -q "Development Tools" && \
-    dnf clean all
+RUN dnf update -y -q --nobest
+RUN dnf clean all
+RUN dnf install -y -q wget unzip which vim python2 python3
+RUN dnf group install -y -q "Development Tools"
+RUN dnf clean all
```

Comment:
Why add the `RUN`? 
each RUN is a layer, and as we are not squashing the layers there is no point in doing a `dnf clean all` as the data remains in the previous layer. 
You can add a comment that explains why they are all in the same instruction.

Reply:
Good point, reverting.